packages/cluster-ui/src/tableStatistics/tableStatistics.tsx

```diff
@@ -54,7 +65,18 @@ export const TableStatistics: React.FC<TableStatistics> = ({
       <h4 className={countTitle}>
         {activeFilters ? resultsCountAndClear : resultsPerPageLabel}
       </h4>
-      <h4 className={lastCleared}>{renderLastCleared(lastReset)}</h4>
+      <div>
```

Comment:
Right now you are floating the children of this div. I think you could get the same effect by adding a "display:flex" to this container and not need to apply styles to the individual children.

Reply:
Done. :+1: